README.md

```diff
@@ -13,12 +13,16 @@ contain a lot more information on `radicle`.
 
 ## Installation
 
-You need [stack](https://docs.haskellstack.org/en/stable/install_and_upgrade/) and `libpq-dev`.
+You will need [`stack`](https://docs.haskellstack.org/en/stable/install_and_upgrade/) `libtinfo-dev`, and `libpq-dev` installed.
```

Comment:
This is quite Ubuntu-specific, and also duplicates some of the info in:

http://docs.radicle.xyz/en/latest/guide/GettingStarted.html#building-from-source

I think we should make the info at that location better, and link to there from the readme.

Reply:
sounds reasonable. should I close this PR then?

Reply:
No, just move the edits to this file instead: `docs/source/guide/GettingStarted.rst`.

Reply:
cool, just moved.